src/components/learner-credit-management/cards/NewAssignmentModalButton.jsx

```diff
@@ -57,6 +76,25 @@ const NewAssignmentModalButton = ({ course, children }) => {
     setCanAllocateAssignments(canAllocate);
   }, []);
 
+  const onSuccessEnterpriseTrackEvents = () => {
+    sendEnterpriseTrackEvent(
+      enterpriseId,
+      EVENT_NAMES.LEARNER_CREDIT_MANAGEMENT.CLOSE_ASSIGNMENT_MODAL_ASSIGNED,
+      {
+        isOpen: !isOpen,
+        assignButtonState,
+      },
+    );
```

Comment:
There was an additional track event in `handleCloseAssignmentModal` which I forgot to refactor out actually 👍🏽  great catch on that. (Opted for distinct events to handle exit and close instead of generalizing the solution).

I like the idea of consolidating the events and will give it a refactor now. 💯